generated/python/gapic-google-cloud-error-reporting-v1beta1/google/cloud/gapic/errorreporting/v1beta1/error_stats_service_client.py

```diff
@@ -33,14 +33,14 @@
 import google.gax
 
 from google.cloud.gapic.errorreporting.v1beta1 import enums
-from google.devtools.clouderrorreporting.v1beta1 import error_stats_service_pb2
+from google.cloud.grpc.devtools.clouderrorreporting.v1beta1 import error_stats_service_pb2
```

Comment:
The gRPC path should be `google.cloud.grpc.errorreporting.v1beta1`, which is analogous to the GAPIC path of `google.cloud.gapic.errorreporting.v1beta1`.